content/en/dynamic_instrumentation/expression-language.md

```diff
@@ -37,6 +37,16 @@ The following sections summarize the variables and operations that the Dynamic I
 
 **Note**: These variables are **only available** when instrumenting an **entire method**, and **not** for individual lines of code.
 
+## Special variables
```

Comment:
`@return` etc can also be considered a "special" variable, and `@it` etc can also be considered a contextual variable. So these two headings are a bit ambiguous.

I think I'd group these two sections under a common "Special variables" heading and then have two level 3 headings called, for example, "Method instrumentation variables" and "Collection and dictionary variables".

Reply:
there is also `@exception` and `@duration` :)

Reply:
Okay, let me collapse these two sections into a single table